### PR TITLE
Feature/115 holiday calculator plugin

### PR DESCRIPTION
**Pull request by ckunki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ A time recording tool
 ### Notes
 
 * Won't work on weekends. To force working on a weekend, manually change the day type to `WORK`.
-* Public holidays are not detected automatically. Set the day type to `HOLIDAY` manually.
+* Public holidays can be calculated using plugin [holidays-calculator](README.md#holidays_calculator) or user can are set the day type to `HOLIDAY` manually.
```
Comment (kaklakariada) [inline README.md:52]: 👍

File: plugins/csv/src/test/java/org/itsallcode/whiterabbit/plugin/csv/CSVExporterTest.java [hunk 2]
```diff
@@ -67,39 +76,54 @@ void exportMultipleProjectsPerDay()
...
-        assertThat(tmpOutStream).hasToString("Date,Project,TimePerProject,TimePerDay,Comment\n2021-06-04,,,07:00,day comment\n,Project9FromOuterSpace,01:00,,abc\n,other_project,03:00,,\n,,03:00,,\n");
+        assertThat(tmpOutStream).hasToString(lines(
+                "Date,Project,TimePerProject,TimePerDay,Comment",
+                "2021-06-04,,,07:00,day comment",
+                ",Project9FromOuterSpace,01:00,,abc",
+                ",other_project,03:00,,",
+                ",,03:00,,"));
+    }
+
+    private static String lines(String... elements)
```
Comment (kaklakariada) [inline plugins/csv/src/test/java/org/itsallcode/whiterabbit/plugin/csv/CSVExporterTest.java:89]: ❤️

File: plugins/holiday-calculator/build.gradle [hunk 3]
```diff
@@ -0,0 +1,8 @@
+dependencies {
+	implementation 'org.itsallcode:holiday-calculator:0.1.0'
+
+    runtimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}") {
```
Comment (kaklakariada) [inline plugins/holiday-calculator/build.gradle:4]: Should only be required for tests?
```suggestion
    testRuntimeOnly("org.apache.logging.log4j:log4j-slf4j-impl:${log4jVersion}") {
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]
Thank you very much for your pull request!
Only a minor finding :)

Code duplication accepted for now, we can fix this later.

File: plugins/holiday-calculator/build.gradle (hunk 3 above)
Comment (ckunki) [reply plugins/holiday-calculator/build.gradle:4]: I'm not sure. 
In main-runtime logging seems fine - maybe a logger is provided from somewhere else? 
If so, then yes, you're right: only in test I noticed a logger to seem to have been missing.

**Review by kaklakariada — APPROVED** [review]

**Comment by ckunki** [comment]
resolved code duplication.

**Comment by kaklakariada** [comment]
Sorry, i already merged the pr. Please create another pr.

**Pull request merged**